test/multiverse/suites/view_component/Envfile

```diff
@@ -0,0 +1,20 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+instrumentation_methods :chain, :prepend
+
+VIEW_COMPONENT_VERSIONS = [
+  [nil, 2.7],
+  ['2.0.0', 2.4]
+]
+
+def gem_list(view_component_version = nil)
+  <<-RB
```

Comment:
Why did you choose to use `<<-` over `<<~`? Do you want the leading whitespace?

Reply:
No good reason lol I think it might have been a copy & paste. I've updated https://github.com/newrelic/newrelic-ruby-agent/pull/2367/commits/9fa2fb0d30b2ed490f571515083ba0da09a0cbe3